src/utils/i18n/locales/en/en.json

```diff
@@ -0,0 +1,131 @@
+{
+
+  "navbar":{
+    "title": "React ChatBotify Gallery",
+    "plugin": "Plugins",
+    "themes": "Themes",
+    "themebuild": "Theme Builder",
+    "docs": "Documentation",
+    "discord": "Discord",
+    "login": "Login",
+    "logout": "Logout",
+    "lang": "English",
+    "profile": "Profile",
+    "light": "Light",
+    "dark": "Dark"
+  },
+
+  "home":{
+    "greet":"Welcome to React ChatBotify Gallery",
+    "description":"Browse, rate and share themes for your chatbot today!",
+    "login":"Login",
+    "browse":"Browse Themes",
+    "welcome":"It's good to see you"
+  },
+
+  "themes":{
+    "search": "Search",
+    "searchbar":"Search Themes...",
+
+    "thcreds":{
```

Comment:
Let's try to avoid ambiguous keys, prefer it to be readable over keeping it too abbreviated. Same for other abbreviations below.